.gitignore

```diff
@@ -46,3 +46,6 @@ Thumbs.db
 ##############
 Build/tools/
 .buildsettings
+
+Source/UserManagement/Web.Angular/src/app/DataCollectors/**/*.js
+Source/UserManagement/Web.Angular/src/app/GreetingGenerators/**/*.js
```

Comment:
We cannot ignore these folders, the frontend code will sit next to the proxy code, so we cannot ignore these folders

Reply:
Good point @woksin - since it would require people to have to run DotNET build on the Web project to get the proxies out before you can start doing your frontend. They're in totally different folders / projects. Not everyone has .NET installed even, if their focus is on the frontend Web side.